src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -5066,27 +5066,47 @@ private void OnDecorationBoundingBoxChanged(int x, int y, int width, int height)
         }
         private void OnInputColorChanged(float x, float y, float z, float w)
         {
-            InputColor = new Vector4(x, y, z, w);
+            ReusablePool<Vector4>.GetOne((vector4, x, y, z, w) =>
+            {
+                vector4.Reset(x, y, z, w);
+                SetInternalInputColor(vector4);
+            }, x, y, z, w);
```

Comment:
I already explained how to fix this in my previous comment.
```suggestion
            ReusablePool<Vector4>.GetOne((vector4, x, y, z, w) =>
            {
                vector4.Reset(x, y, z, w);
                SetInternalInputColor(vector4);
                NotifyPropertyChanged("InputColor");
            }, x, y, z, w);
```

Reply:
Got it. Thank you very much for detailed explanation.
But I will remove ReusablePool as original, because setter in TextEditor is called only when obsolete APIs are called.
obsolete APIs are called not frequently, I think.